ci/builders/mac_ios_engine.json

```diff
@@ -120,6 +120,132 @@
                     "sdk_version": "14e300c"
                 }
             }
+        },
+        {
+            "drone_dimensions": [
+                "device_type=none",
+                "mac_model=Macmini8,1",
```

Comment:
Do they have to run on x64 machines? if not I'd recommend to remove this line here and below.

Reply:
I copied this from other above jobs. I thought the jobs for arm64 builds can be run on arm machines, but the ones without arm64 (like this one) should not. 
I can fix the existing ones in a separate PR then fix this too.

Reply:
Move all the jobs to arm machines except the x86 simulator